staging/src/k8s.io/apiserver/pkg/endpoints/filters/impersonation.go

```diff
@@ -104,14 +104,14 @@ func WithImpersonation(handler http.Handler, a authorizer.Authorizer, s runtime.
 				userExtra[extraKey] = append(userExtra[extraKey], extraValue)
 
 			default:
-				klog.V(4).Infof("unknown impersonation request type: %v", impersonationRequest)
+				klog.V(4).InfoS("unknown impersonation request type", "Request", impersonationRequest)
 				responsewriters.Forbidden(ctx, actingAsAttributes, w, req, fmt.Sprintf("unknown impersonation request type: %v", impersonationRequest), s)
 				return
 			}
 
 			decision, reason, err := a.Authorize(ctx, actingAsAttributes)
 			if err != nil || decision != authorizer.DecisionAllow {
-				klog.V(4).Infof("Forbidden: %#v, Reason: %s, Error: %v", req.RequestURI, reason, err)
+				klog.ErrorS(err, "Forbidden", "URI", req.RequestURI, "Reason", reason)
```

Comment:
this should remain v(4).

Reply:
updated the code and the logs is  belowing.
before:
I0120 10:30:02.553180     388 impersonation.go:114] Forbidden: "/", Reason: , Error: works on my machine
after:
I0120 10:30:02.553180     388 impersonation.go:115] "Forbidden" URI="/" Reason="" Error="works on my machine"